cmd/queue/main.go

```diff
@@ -133,26 +134,49 @@ var (
 		stats.UnitMilliseconds)
 )
 
-func initEnv() {
-	containerConcurrency = util.MustParseIntEnvOrFatal("CONTAINER_CONCURRENCY", logger)
-	queueServingPort = util.MustParseIntEnvOrFatal("QUEUE_SERVING_PORT", logger)
-	revisionTimeoutSeconds = util.MustParseIntEnvOrFatal("REVISION_TIMEOUT_SECONDS", logger)
-	servingConfig = util.GetRequiredEnvOrFatal("SERVING_CONFIGURATION", logger)
-	servingNamespace = util.GetRequiredEnvOrFatal("SERVING_NAMESPACE", logger)
-	servingPodIP = util.GetRequiredEnvOrFatal("SERVING_POD_IP", logger)
-	servingPodName = util.GetRequiredEnvOrFatal("SERVING_POD", logger)
-	servingRevision = util.GetRequiredEnvOrFatal("SERVING_REVISION", logger)
-	servingService = os.Getenv("SERVING_SERVICE") // KService is optional
-	userTargetPort = util.MustParseIntEnvOrFatal("USER_PORT", logger)
+type envConfig struct {
+	ContainerConcurrency   string `envconfig:"CONTAINER_CONCURRENCY" required:"true"`
```

Comment:
Do you need to define the key everywhere here? I think the library converts `ContainerConcurrency` into `CONTAINER_CONCURRENCY` automatically.

Reply:
yes i have used ```split_words:"true"```